From 0edb343ea86b1b5145c2ecbaefe2de67c539db39 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Thu, 25 Dec 2025 10:16:33 +0100
Subject: [PATCH 2/4] Add GcamGo and LatinIME to product package

Add Gcam Go to product package - Version HDR
LatinIME to support slice keyboard

Change-Id: I5d9c09a65525828f4d771ccb520a7ab1aaad3a8e
---
 target/product/handheld_product.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index b2ce4c5e19..f54bf55732 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -25,6 +25,8 @@ PRODUCT_PACKAGES += \
     Browser2 \
     Calendar \
     Contacts \
+    GcamGo \
+    LatinIME \
     DeskClock \
     Gallery2 \
     Music \
-- 
2.25.1

